### ISSUE TITLE
Install fails on nodegit 404

### ISSUE THREAD

**Issue by ubuntupunk** [dreamsicle-io/create-wp-theme#28]
```
npx @dreamsicle.io/create-wp-theme my-theme
[############......] / extract:lodash: sill extract lodash@^4.17.14 extracted to /home/vagrant/.npm/_[############......] / 
extract:lodash: sill extract lodash@^4.17.14 extracted to /home/vagrant/.npm/_npx/29029[############......] / extract:lodash: 
sill extract lodash@^4.17.14 extracted to /home/vagranode-pre-gyp ERR! install response status 404 Not Found on https://
axonodegit.s3.amazonaws.com/nodegit/nodegit/nodegit-v0.28.0-alpha.23-node-v83-linux-x64.tar.gz
gyp ERR! configure error
gyp ERR! stack TypeError: Cannot read property 'pipeline' of undefined
gyp ERR! stack at Object.<anonymous> (/home/vagrant/.npm/_npx/29029/lib/node_modules/@dreamsicle.io/create-wp-theme/node_modules/node-gyp/lib/install.js:8:32)



```

**Comment by theenoahmason** [comment]
@ubuntupunk Sorry for just get around to this. I cannot reproduce your issue, but I did just release some small updates to the CLI and some large updates the the WP Theme Assets package (that this script outputs). I just ran it and it completed successfully. Could you provide some information on your machine? We use this script internally at our agency to scaffold new projects but would love to help get you going if you find it useful.

I am seeing that nodegit is getting a bit old. Going to take a look into replacing it but would love to know why it didnt work for you.

**Comment by theenoahmason** [comment]
@ubuntupunk I have just released #35 which replaced `nodegit` with a native `git clone` call using `child_process` and published to npm 👍

This will fix your issue and removes a large dep from the project as an added bonus. Closing this issue now as `nodegit` is no longer part of the project. Please create a new issue if you continue to have problems.
